Jenkinsfile

```diff
@@ -86,6 +86,41 @@ try {
                         stage('Build') {
                             sh "${params}"
                         }
+                        
+                        set_gh_status_pending(is_pr, 'Testing with QEMU')
+                        params = ["${script_env_global}", "${script_env_local}",
+                                  "docker/test-qemu.sh"].join("\n")
+                        stage("Test QEMU") {
+                            try{
+                                sh "${params}"
+                            } catch (Exception e) {
+                                throw e
+                            } finally{
+                                archiveArtifacts allowEmptyArchive: true,
```

Comment:
should we rename TEST-* files from QEMU testing similar to renaming after HW-based testing? Otherwise QEMU test results files will be named just TEST-oeqa.runtime-* and it remains unclear on which were these produced.